server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -167,6 +167,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion DESIRED_NODE_VERSION_OPTIONAL_STRING = def(8_580_00_0);
     public static final TransportVersion ML_INFERENCE_REQUEST_INPUT_TYPE_UNSPECIFIED_ADDED = def(8_581_00_0);
     public static final TransportVersion ASYNC_SEARCH_STATUS_SUPPORTS_KEEP_ALIVE = def(8_582_00_0);
+    public static final TransportVersion NORMALISED_BYTES_PARSED = def(8_583_00_0);
```

Comment:
Could this be more descriptive? eg `INDEX_REQUEST_NORMALIZED_BYTES_PARSED`? It's a mouthful, but it gives at least some context as to what the normalized bytes are for.